support/camel-k-maven-plugin/src/main/java/org/apache/camel/k/tooling/maven/GenerateCatalogMojo.java

```diff
@@ -526,6 +545,18 @@ private void addCapabilities(RuntimeSpec.Builder runtimeSpec, CamelCatalogSpec.B
         artifacts.add(Artifact.from("org.apache.camel.quarkus", "camel-quarkus-jaxb"));
         artifacts.add(Artifact.from("org.jolokia", "jolokia-agent-jvm", "javaagent"));
         addCapabilityAndDependecies(runtimeSpec, catalogSpec, "jolokia", artifacts, true);
+
+        properties.clear();
+        properties.add(Property.from("level", "quarkus.log.level"));
+        properties.add(Property.from("color", "quarkus.console.color"));
+        properties.add(Property.from("format", "quarkus.log.console.format"));
+        properties.add(Property.from("json", "quarkus.log.console.json"));
+        properties.add(Property.from("jsonPrettyPrint", "quarkus.log.console.json.pretty-print"));
```

Comment:
I don't see a json logging option in spring-boot https://docs.spring.io/spring-boot/docs/3.0.0/reference/htmlsingle/#features.logging, it seems that to enable json logging one has to use a backend specific logging configuration

Reply:
Correct. Same comment as with the above telemetry capability.